o2g/osm/builders/stations.md

```diff
@@ -0,0 +1,37 @@
+Notes on Stations
```

Comment:
I think you should consider putting this documentation inside the python code. I think this is bound to get outdated or overlooked otherwise. Just an idea though ;)

Reply:
Hmm, if it fits in one module I'll move it inside.